pl_examples/basic_examples/multi_node_ddp2_demo.py

```diff
@@ -4,23 +4,19 @@
 import os
 from argparse import ArgumentParser
 
-import numpy as np
-import torch
-
 import pytorch_lightning as pl
 from pl_examples.models.lightning_template import LightningTemplateModel
 
-SEED = 2334
-torch.manual_seed(SEED)
-np.random.seed(SEED)
+pl.seed_everything(234)
+
 
+def main(model_args):
+    """ Main training routine specific for this project """
 
-def main(hparams):
-    """Main training routine specific for this project."""
     # ------------------------
     # 1 INIT LIGHTNING MODEL
     # ------------------------
-    model = LightningTemplateModel(hparams)
+    model = LightningTemplateModel(model_args)
```

Comment:
Should this not be: `model = LightningTemplateModel(**vars(model_args))` similar to the other scrips?

Reply:
very good, thanks. feel free to make it a suggestion so we can commit it together :)